charts/handbrake/0.0.1/test_values.yaml

```diff
@@ -0,0 +1,91 @@
+image:
+  repository: jlesage/handbrake
+  tag: v1.23.1
+  pullPolicy: IfNotPresent
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 5800            #Required for web GUI
+appAdditionalServicesEnabled: true
+appAdditionalServices:
+  vnc:
+    enabled: true
+    type: ClusterIP
+    port:
+      port: 5900
+
+
+# Path to your optical drive device in the container
+additionalVolumeMounts: []
+#  - name: opticaldrive
+#    mountPath: /dev/serial/by-id/sr0-
+
+# Path to your zwave device on the host
+additionalVolumes: []
+#  - name: opticaldrive
+#    hostPath:
+#      path: /dev/serial/by-id/sr0-
+
+# affinity:
+#   nodeAffinity:
+#     requiredDuringSchedulingIgnoredDuringExecution:
+#       nodeSelectorTerms:
+#       - matchExpressions:
+#         - key: app
+#           operator: In
+#           values:
+#           - opticaldrive-controller???  #zwave-controller
+
+
+
+#All values here are set as the docker defaults.
+env:
+# Permissions Settings
```

Comment:
Please check if this is working with common 1.2.0

Reply:
Made a new app install and pointed to previously created folders with files that had 1000:1000 UID and GID. 
All Folders and contents now have 568:568 UID and GID, except config that has 1000:1000. 
Probably because handbrake reapplies permissions on launch on its own configs. 1000:1000 is what is set on values.yaml.